cdk-custom-resource-multiple-lambdas/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Custom resource with multiple Lambda functions",
+  "description": "This pattern demonstrates the use of multiple Lambda functions to manage a custom resource with CDK.",
+  "language": "Python",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "AWS CloudFormation can wait for up to an hour for a custom resource to report its success or failure. However, AWS Lambda function execution is limited with just 15 minutes maximum.",
+      "This sample project demonstrates how to use multiple AWS Lambda functions to manage a custom resource, so that developers are not limited with 15 minutes Lambda execution timeout.",
```

Comment:
```suggestion
      "This sample project demonstrates how to use multiple AWS Lambda functions to manage a custom resource, so that developers are not limited by the 15 minutes Lambda execution timeout.",
```